var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -138,7 +139,10 @@ def install(self, spec, prefix):
         args = self.cmake_args()
         args.extend(std_cmake_args)
 
-        source_directory = join_path(self.stage.source_path, "cmake")
+        if self.spec.satisfies("@:3"):
```

Comment:
```suggestion
        if self.spec.satisfies("@:3.20"):
```
The root CMakeLists.txt didn't exist until 3.21. Honestly, we should just remove v2.5.0 and convert this to a `CMakePackage` at some point, but that doesn't have to happen in this PR if you don't want to.

Reply:
Done. I don't know how to properly combine `@when` and `@property`, please correct me if there is a more spack-ish way.